### PR TITLE
Add meta/main.yml

### PR DESCRIPTION
**Pull request by cognifloyd**
This makes installing via ansible-galaxy possible.

### CODE REVIEW

File: meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  platforms:
+    - name: EL
+      versions:
+        - all
+    - name: Fedora
+      versions:
+        - all
+  galaxy_tags: []
+dependencies:
+  - jtyr.config_encoder_filters
```
Comment (jtyr) [inline meta/main.yml]: Please change this like to:

```
- config_encoder_filters
```

and follow [installation instructions](https://github.com/jtyr/ansible-config_encoder_filters#installation) to install the dependency.

File: meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+
+galaxy_info:
+  author: Jiri Tyr
+  description: Ansible role creates RPM build environment
```
Comment (jtyr) [inline meta/main.yml]: Please change this line to:

```
description: Ansible role which creates RPM build environment.
```

File: meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  license: MIT
+  min_ansible_version: 1.9
+  platforms:
+    - name: EL
+      versions:
+        - all
+    - name: Fedora
```
Comment (jtyr) [inline meta/main.yml]: Is it necessary to specify Fedora if all EL versions are covered?

**Review by jtyr — CHANGES_REQUESTED** [review]

**Comment by cognifloyd** [comment]
So, I've returned to the project where I'm using rpmbuild after a year on other stuff :) I resolved the issues you mentioned.

**Comment by jtyr** [comment]
This has been implemented.

**Pull request closed without merging**